src/components/Card/Card.jsx

```diff
@@ -10,26 +11,36 @@ import {
 import PropTypes from 'prop-types'
 
 const Card = ({ item }) => {
-  const { main_image, category, name, price } = item.data
-  const { url, alt } = main_image
+  const { id, data } = item
+  const { mainimage, category, name, price } = data
+  const { url, alt } = mainimage
```

Comment:
when restructuring objects you can also do it with nested objects, it depends on preference

```suggestion
  const {
    id,
    data: {
      category,
      name,
      price,
      mainimage: { url, alt },
    },
  } = item
```

Reply:
Thanks! I like this suggestion, I'll apply this change in the next PR.